### ISSUE TITLE
Missing configuration or bug with path overlap

### ISSUE THREAD

**Issue by DenisRozhko** [cezerin/cezerin#621]
admin.production.js and admin.js have some differences in urls. As you see there are absolute path and relative. After setting up cezerin on Ubuntu 18 rely on guid I see the problem that when I try to log in 
as admin it takes store url not API url.

![defect_mail](https://user-images.githubusercontent.com/48088310/53559999-178e3c80-3b54-11e9-8193-1d5ba71f0aa0.jpg)

When I hardcoded 3001 port it works well.

Thank you in advance,
Denys

**Comment by ngoyal16** [comment]
this repo is dead you can contact to the new repo at

https://github.com/Cezerin2

also close this issue too

**Comment by dewelloper** [comment]
@DenisRoshko yes you are right. That is working right when you set port to 3001 at http. But if you make it on https ıts not working as expected. Have you tryed for https ?
